### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [epidrome/cover-card#16]
The user @carlsonsantana validate your site template "https://epidrome.github.io/cover-card/" and found these accessibility errors:
* Page https://epidrome.github.io/cover-card/:
  * Issue *d3c8c6d5-71d4-4c2c-b9f0-187177aab628*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
    * **Message**: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
    * **Selector**: `#avatar`
    * **Context**: `<img src="avatar.jpg" id="avatar">`
  * Issue *a98c0aa6-faee-4ab4-a510-4b220a1d62c4*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
    * **Message**: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
    * **Selector**: `#fullscreen`
    * **Context**: `<img src="background.jpg" id="fullscreen">`
  * Issue *83ae4206-3fc1-4400-9f5a-fdfcca6255b2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
        <meta charset="u...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
